### PR TITLE
Changing logging level

### LINKED ISSUE #2377: Export failure should be logged
When an exporting compile fails in the CompileServer, this should be  logged as warning, as this is usually a sign of a serious problem

### PR DESCRIPTION
**Pull request by edvgui**
# Description

Changing logging level for compile export failure in CompilerService.

closes #2377 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] ~~Changelog entry~~
- [x] ~~Type annotations are present~~
- [x] ~~Code is clear and sufficiently documented~~
- [x] ~~No (preventable) type errors (check using make mypy or make mypy-diff)~~
- [x] ~~Sufficient test cases (reproduces the bug/tests the requested feature)~~
- [x] Correct, in line with design
- [x] ~~End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )~~

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

File: src/inmanta/server/services/compilerservice.py [hunk 1]
```diff
@@ -276,7 +276,7 @@ async def run(self, force_update: Optional[bool] = False) -> Tuple[bool, Optiona
             result = await self._run_compile_stage("Recompiling configuration model", cmd, project_dir, env=env_vars_compile)
             success = result.returncode == 0
             if not success:
```
Comment (wouterdb) [inline src/inmanta/server/services/compilerservice.py:278]: you missed one important word: exporting compile 

when self.request.do_export is false, it can stay debug.

**Review by wouterdb — CHANGES_REQUESTED** [review]

File: src/inmanta/server/services/compilerservice.py [hunk 2]
```diff
@@ -276,7 +276,7 @@ async def run(self, force_update: Optional[bool] = False) -> Tuple[bool, Optiona
...
-                LOGGER.debug("Compile %s failed", self.request.id)
+                LOGGER.warning("Compile %s failed", self.request.id)
```
Comment (sanderr) [inline src/inmanta/server/services/compilerservice.py:279]: The issue description seems to state that this should only be a warning for exporting compiles `self.request.do_export is True`. I'm not sufficiently familiar with the use case to be sure. @wouterdb you created the ticket, what was the intended behavior?

File: .gitignore [hunk 3]
```diff
@@ -46,6 +46,7 @@ src/*/parser/parsetab.py
 
 # Venv
 .env
+venv
```
Comment (sanderr) [inline .gitignore:49]: Just curious, why did you add this? How does this directory get created?

**Review by sanderr — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (edvgui) [reply .gitignore:49]: This is where I put the venv i create manually, I might have to to keep that change on my machine, it doesn't concern anyone else I suppose

Comment (edvgui) [reply .gitignore:49]: (Yes I still create virtualenv manually, for some reason virtualenvwrapper wouldn't create me environment with the right python version, and given that I don't find it too painful to make the commands manually, I continue that way)

Comment (sanderr) [reply .gitignore]: All right. I'm not against merging this in. A directory named `venv` in the root of the project is a sufficiently common scenario to allow it in the gitignore in my opinion.
Have you tried using the `-p` parameter to specify a python version? `mkvirtualenv -p python3.6 myenv`.

File: src/inmanta/server/services/compilerservice.py (hunk 1 above)
Comment (edvgui) [reply src/inmanta/server/services/compilerservice.py:278]: Changed it in https://github.com/inmanta/inmanta/pull/2475/commits/1c98e58913d8f09979868d60e98213abdd540a66

File: src/inmanta/server/services/compilerservice.py (hunk 2 above)
Comment (edvgui) [reply src/inmanta/server/services/compilerservice.py]: Changed it in https://github.com/inmanta/inmanta/pull/2475/commits/1c98e58913d8f09979868d60e98213abdd540a66

**Review by sanderr — APPROVED** [review]

File: .gitignore (hunk 3 above)
Comment (edvgui) [reply .gitignore]: Revert in https://github.com/inmanta/inmanta/pull/2475/commits/ea3ff86634f5f1e0c5a133527a3a9a5da05c6f3f

**Review by wouterdb — APPROVED** [review]

File: .gitignore (hunk 3 above)
Comment (edvgui) [reply .gitignore]: I already tried that yes, but for some reason the python version was still 3.8, I didn't search really long what was going wrong to be honest, and now with the advice of bart I will not need those venv anymore
